install.sh

```diff
@@ -1,27 +1,102 @@
 #!/bin/bash
+echo -n "Enter your github email: "
+read email
+
+if [ -f "$HOME/.ssh/id_rsa.pub" ]; then
+   sudo cp $HOME/.ssh/id_rsa.pub $HOME/sshkey.txt
+else
+   ssh-keygen -t rsa -b 4096 -C $email
+   eval "$(ssh-agent -s)"
+   ssh-add $HOME/.ssh/id_rsa
+   sudo cp $HOME/.ssh/id_rsa.pub $HOME/sshkey.txt
+fi
+echo ""
+echo $(cat $HOME/sshkey.txt)
+echo -e "\nssh public key: $HOME/sshkey.txt\n"
+echo -e "=======> https://github.com/settings/ssh/new\n"
+
+echo "Did you register the key?"
+sleep 3
+select ans in "Yes" "No"; do
+   case $ans in
+      Yes )
+         break;;
+      No )
+         exit;;
+   esac
+done
+eval `ssh-agent`
+ssh-add
+
+check=$(grep -E '^(VERSION|NAME)=' /etc/os-release)
+arch='Arch'
+ubuntu='Ubuntu'
+fedora='Fedora'
+debian='Debian'
+
+if [[ $check == *$arch* ]]; then
```

Comment:
Also, for every distro, add:
```
git-lfs install
```
This is necessary to be able to pull simulation files.